plugin/mutate/source/dextool/plugin/mutate/backend/report/html/package.d

```diff
@@ -564,23 +564,38 @@ struct MetaSpan {
         noCoverage,
     }
 
+    static struct MutationLength {
+        ulong length;
+        MutationId id;
+        Mutation.Status status = Mutation.Status.killed;
+    }
+
     StatusColor status;
     string onClick;
+    MutationLength clickPrio;
 
     this(const(FileMutant)[] muts) {
         immutable click_fmt2 = "ui_set_mut(%s)";
         status = StatusColor.none;
-
+        //init
+        if (muts.length != 0) {
+            clickPrio.length = muts[0].txt.mutation.length;
+            clickPrio.id = muts[0].id;
+            clickPrio.status = muts[0].mut.status;
+        }
         foreach (ref const m; muts) {
             status = pickColor(m, status);
-            if (onClick.length == 0 && m.mut.status == Mutation.Status.alive) {
-                onClick = format(click_fmt2, m.id.get);
+            if (m.mut.status == clickPrio.status
```

Comment:
Need to impl the previous algo:


1. priorities alive mutants over everything else
2. priorities the shortest alive
3. priorities shortest mutant (which mean that if there are no alive mutants on the line then it is only the size)